debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresOffsetContext.java

```diff
@@ -135,6 +135,11 @@ public void updateWalPosition(Lsn lsn, Lsn lastCompletelyProcessedLsn, Instant c
         sourceInfo.update(lsn, commitTime, txId, tableId, xmin);
     }
 
+    public void updateWalPosition(Lsn lsn, Lsn lastCompletelyProcessedLsn, Instant commitTime, Long txId, Long xmin) {
```

Comment:
This was needed because the `updateWalPosition` function defined above takes the `tableId` argument and passes it to `sourceInfo.update()` which does not respect the null value for the `tableId`. In tests, we were seeing these messages having a defined tableId. 

https://github.com/debezium/debezium/blob/fe26649b4b5eca179fa5e0a2f2511e1efd49f8e1/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java#L114-L128

This is the new `update()` function that was added to `SourceInfo` to explicitly set the `tableName` and `schemaName` to `null`

https://github.com/debezium/debezium/blob/fe26649b4b5eca179fa5e0a2f2511e1efd49f8e1/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java#L150-L158

That was the original thought process, but this may no longer be needed if we set the `schemaName` and `tableName` to `N/A` for these messages.

Reply:
So in order to set the `tableName` and `schemaName` to an empty string or "N/A" without using these overloaded functions, I passed 
`TableId tableId = PostgresSchema.parse("N/A.N/A")` and passed that to this `updateWalPosition` function here https://github.com/debezium/debezium/blob/fe26649b4b5eca179fa5e0a2f2511e1efd49f8e1/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java#L246

While that technically works and allows us to get rid of these overloaded functions, it feels a bit "hacky". Another option is to keep the overloaded functions then set the `tableName` and `schemaName` directly in the `SourceInfo` class. These new overloaded functions could also be used for other potential message types that may not be associated with a table.  Any thoughts on those two different approaches?